### PR TITLE
fix up whitespace

### PR DESCRIPTION
**Pull request by tacaswell**
@danielballan can you look at this?

### CODE REVIEW

File: 2016-08-31/roster.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # Roster
-modified!
```
Comment (tacaswell) [inline 2016-08-31/roster.md:2]: Why was this removed?

**Comment by kgofron** [comment]
Sounds good
